### ISSUE TITLE
Split financing of services

### ISSUE THREAD

**Issue by Fohlen** [inexorgame/inexor-core#563]
I've paid the last year of Inexor services, all expenses can be viewed at https://opencollective.com/inexorgame/#budget

Now is the time again where I will have to "refresh" our uberspace account so it will run further.
This ticket aims to solve 2 problems

- solve financing problems (who should pay what or not)
- discuss wether we actually need uberspace

Currently only Inexor stuff runs on our uberspace (everything else has been migrated since Aug/2017). Namely the following services

- building and serving the website from git
- git-mirror of the wiki
- git mirror of grpc-web (disabled because grpc-web is going beta)
- aggregator.inexor.org (disabled because nobody uses it currently)
- emails

Please let me know how we could resolve this.
For the services:

- the website could be done with GitHub pages & Travis
- free cronjob services exist for mirroring git repositories
- emails will be need to be hosted somewhere..

**Comment by IAmNotHanni** [comment]
I would like to financially support Inexor.

**Comment by Fohlen** [comment]
Yes. See the receipt attached on open collective. It's for one year (1.1.17-1.1.18)

**Comment by Croydon** [comment]
30€ / year for uberspace hosting
12€ / year for domain
= 42€ / year

**Comment by Fohlen** [comment]
It's actually way more, `15,35` currently. The domain price can be checked at https://www.hetzner.de/domainregistration

**Comment by Croydon** [comment]
I paid 15,35€ / 24 € remaining.

**Comment by Fohlen** [comment]
Update to this ticket: we abandoned `uberspace` for now, so the last cost to remain is `inexor.org` domain.

**Comment by Fohlen** [comment]
Update for 2019. I have paid the 2019 cost (12,90). Also added this to OpenCollective.
